package.json

```diff
@@ -33,7 +33,8 @@
     "build-storybook": "yarn ./storybook run build-storybook",
     "techdocs-cli": "node scripts/techdocs-cli.js",
     "techdocs-cli:dev": "cross-env TECHDOCS_CLI_DEV_MODE=true node scripts/techdocs-cli.js",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "husky install"
```

Comment:
See https://typicode.github.io/husky/#/?id=yarn-2 - this is the hook name used for yarn 2+, `prepare` isn't the one to use for Yarn 3 apparently